src/VuetifyResource.vue

```diff
@@ -237,6 +239,12 @@
           <template slot="no-results">
             {{ lang('no-results') }}
           </template>
+          <template v-for="(_, name) in $slots" v-slot:[name]>
+            <slot :name="name"/>
+          </template>
+          <template v-for="(_, name) in $scopedSlots" :slot="name" slot-scope="slotData">
+            <slot :name="name" v-bind="slotData"/>
+          </template>
```

Comment:
Dit is deprecated :) https://vuejs.org/v2/guide/components-slots.html#Deprecated-Syntax

v-slot is de nieuwe